aws/resource_aws_apigatewayv2_integration_test.go

```diff
@@ -0,0 +1,451 @@
+func testAccAWSAPIGatewayV2IntegrationConfig_integrationTypeHttp(rName string) string {
+	return testAccAWSAPIGatewayV2IntegrationConfig_apiWebSocket(rName) + fmt.Sprintf(`
+resource "aws_apigatewayv2_integration" "test" {
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "HTTP"
+
+  connection_type               = "INTERNET"
+  content_handling_strategy     = "CONVERT_TO_TEXT"
+  description                   = "Test HTTP"
+  integration_method            = "GET"
+  integration_uri               = "http://www.example.com"
+  passthrough_behavior          = "WHEN_NO_MATCH"
+  template_selection_expression = "$request.body.name"
+  timeout_milliseconds          = 28999
+
+  request_templates = {
+    "application/json" = ""
+  }
+}
+`)
+}
+
+func testAccAWSAPIGatewayV2IntegrationConfig_integrationTypeHttpUpdated(rName string) string {
+	return testAccAWSAPIGatewayV2IntegrationConfig_apiWebSocket(rName) + fmt.Sprintf(`
+resource "aws_apigatewayv2_integration" "test" {
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "HTTP"
+
+  connection_type               = "INTERNET"
+  content_handling_strategy     = "CONVERT_TO_BINARY"
+  description                   = "Test HTTP updated"
+  integration_method            = "POST"
+  integration_uri               = "http://www.example.org"
+  passthrough_behavior          = "WHEN_NO_TEMPLATES"
+  template_selection_expression = "$request.body.id"
+  timeout_milliseconds          = 51
+
+  request_templates = {
+    "application/json" = "#set($number=42)"
+    "application/xml"  = "#set($percent=$number/100)"
+  }
+}
+`)
+}
+
+func testAccAWSAPIGatewayV2IntegrationConfig_lambda(rName string) string {
+	return testAccAWSAPIGatewayV2IntegrationConfig_apiWebSocket(rName) + baseAccAWSLambdaConfig(rName, rName, rName) + fmt.Sprintf(`
+data "aws_caller_identity" "current" {}
+
+resource "aws_lambda_function" "test" {
+  filename      = "test-fixtures/lambdatest.zip"
+  function_name = %[1]q
+  role          = "${aws_iam_role.iam_for_lambda.arn}"
+  handler       = "index.handler"
+  runtime       = "nodejs10.x"
+}
+
+resource "aws_apigatewayv2_integration" "test" {
+  api_id           = "${aws_apigatewayv2_api.test.id}"
+  integration_type = "AWS"
+
+  connection_type               = "INTERNET"
+  content_handling_strategy     = "CONVERT_TO_TEXT"
+  credentials_arn               = "${data.aws_caller_identity.current.arn}"
+  description                   = "Test Lambda"
+  integration_method            = "POST"
+  integration_uri               = "${aws_lambda_function.test.invoke_arn}"
+  passthrough_behavior          = "WHEN_NO_MATCH"
+}
+`, rName)
+}
+
+func testAccAWSAPIGatewayV2IntegrationConfig_vpcLink(rName string) string {
+	return testAccAWSAPIGatewayV2IntegrationConfig_apiWebSocket(rName) + fmt.Sprintf(`
+data "aws_availability_zones" "available" {
+  state = "available"
+
+// https://github.com/terraform-providers/terraform-provider-aws/issues/12412
```

Comment:
#14200 has been merged now, so this is supported

Reply:
OK, thanks; I'll rebase, remove the comment and retest in `us-west-2` (my testing account now has the LAX1 local zone enabled).

Reply:
👍